src/Components/test/E2ETest/Infrastructure/ServerFixtures/ServerFixture.cs

```diff
@@ -37,7 +37,9 @@ protected static string FindSolutionDir()
         private static Dictionary<string, string> FindProjects()
         {
             var solutionDir = FindSolutionDir();
-            return Directory.GetFiles(solutionDir, "*.csproj", SearchOption.AllDirectories)
+            return 
+                Directory.GetFiles(solutionDir, "*.csproj", SearchOption.AllDirectories)
+                .Where(f => !f.Contains("\\ref\\")) // Exclude ref-assembly projects
```

Comment:
Is it OK to use Windows-specific path separators here? I'm unsure whether the scenario applies on other OSes too.

Reply:
I'm gonna replace these things with what we do in MVC (that uses MSBuild and assembly attributes to find the paths for the projects) to get rid of all these shennanigans

Reply:
I've adopted Pranav's solution for this so we should be good.